OpenImis.RestApi/appsettings.Production.json.dist

```diff
@@ -2,9 +2,16 @@
 	"ConnectionStrings": {
 		"IMISDatabase": "Server=Server;Database=IMIS;User ID=User;Password=Password"
 	},
-	"JwtIssuer": "http://openimis.org",
-	"JwtAudience": "http://openimis.org",
-	"JwtExpireDays": 5,
+	"JWT": {
```

Comment:
This section is not used anymore. Also, it contains implementation specific values. Please remove.